pynestml/codegeneration/printers/gsl_variable_printer.py

```diff
@@ -103,4 +103,4 @@ def _print_buffer_value(self, variable: ASTVariable) -> str:
                     var_name += "_" + str(variable.get_vector_parameter())
             return "spike_inputs_grid_sum_[node." + var_name + " - node.MIN_SPIKE_RECEPTOR]"
 
-        return variable_symbol.get_symbol_name() + '_grid_sum_'
+        return "continuous_inputs_grid_sum_[" + variable.get_name().upper() + "]"
```

Comment:
Can we add an assertion for ``variable_symbol.is_continuous_input_port()`` here?